src/App.jsx

```diff
@@ -14,57 +15,67 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
```

Comment:
The initial state for `selectedGood` is set to 'Jam'. Consider setting it to `null` or an empty string to indicate that no item is selected initially, which might be more intuitive for users.